bot/exts/evergreen/status_codes.py

```diff
@@ -21,49 +22,28 @@ async def http_status_group(self, ctx: commands.Context) -> None:
 
     @http_status_group.command(name='cat')
     async def http_cat(self, ctx: commands.Context, code: int) -> None:
-        """Sends an embed with an image of a cat, portraying the status code."""
-        embed = discord.Embed(title=f'**Status: {code}**')
-        url = HTTP_CAT_URL.format(code=code)
-
-        try:
-            HTTPStatus(code)
-            async with self.bot.http_session.get(url, allow_redirects=False) as response:
-                if response.status != 404:
-                    embed.set_image(url=url)
-                else:
-                    raise NotImplementedError
-
-        except ValueError:
-            embed.set_footer(text='Inputted status code does not exist.')
-
-        except NotImplementedError:
-            embed.set_footer(text='Inputted status code is not implemented by http.cat yet.')
-
-        finally:
-            await ctx.send(embed=embed)
+        """Assemble Cat URL and build Embed."""
```

Comment:
This docstring is used in the `.help` documentation, so we should keep it more human friendly.
```suggestion
        """Send a cat version of the requested HTTP status code."""
```